taskcluster/ci/ui-test-apk/kind.yml

```diff
@@ -157,11 +158,11 @@ tasks:
                 GOOGLE_PROJECT: 'moz-focus-android'
     focus-arm-beta:
         attributes:
-            build-type: nightly-firebase
+            build-type: focus-beta-firebase
             shipping-product: focus
         description: 'UI tests on Beta with firebase'
         run-on-tasks-for: [github-push]
-        run-on-git-branches: [main, releases_v]
+        run-on-git-branches: [releases_v]
```

Comment:
Similarly, it doesn't make sense to run `main`, restricting to `releases`

Reply:
Nice! I had had forgotten we supported regular expressions in `run-on-git-branches`:

https://github.com/taskcluster/taskgraph/blob/b93146ec0ba1fc19d25e2eaf02a8fbe1e022bbc8/src/taskgraph/util/attributes.py#L75